app/api/routers/breeze_buddy/merchant_entities/__init__.py

```
@@ -0,0 +1,195 @@
+        403: If user doesn't have access
+        404: If merchant entity not found
+        500: If there's an error retrieving the merchant
+    """
+    return await get_merchant_entity_by_merchant_id_handler(merchant_id, current_user)
+
+
+@router.put("/merchant/{merchant_id}", response_model=MerchantEntityResponse)
+async def update_merchant_entity(
+    merchant_id: str,
+    merchant_data: MerchantEntityUpdate,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)